src/js/components-react/colorPickerWrap.js

```diff
@@ -0,0 +1,120 @@
+import React, {PureComponent} from 'react';
+import debounce from 'lodash/debounce';
+import color from 'react-color/lib/helpers/color';
+
+export const ColorWrap = (Picker) => {
+  const COLOR_EMPTY = {
+    hex: '',
+    hsl: {a: '',
+      h: '',
+      l: '',
+      s: '',
+    },
+    hsv: {a: '',
+      h: '',
+      s: '',
+      v: '',
+    },
+    oldHue: '',
+    rgb: {a: '',
+      b: '',
+      g: '',
+      r: '',
+    }
+  };
+  class ColorPicker extends PureComponent {
+
+    constructor(props) {
+      super();
+      if (props.hasOwnProperty('color')) {
+        this.state = {
+          ...color.toState(props.color, 0),
+        };
+      } else {
+        this.state = COLOR_EMPTY;
+      }
```

Comment:
This function is deprecated and is react anti-pattern so please, refactor this code.